### DISCUSSION TITLE
Horizon Theme

### DISCUSSION THREAD

**Discussion by osaket** [swellstores/community#20] [Help] [answered]
Hi,

I'm wondering when Horizon Theme will be released? And what type of front-end stack will be used for the website and the quiz builder? It's been overdue to release the theme since Q1 this year.

Cheers

**Comment by logeshswell** [comment]
@ana-estrela Could you please share the latest update here?

**Comment by ana-estrela** [comment, marked as answer]
Hey @osaket 👋 

We're still actively working on Horizon's release. It has taken us longer than we wanted because Horizon is dependent on some changes to the storefront dashboard and the way the platform handles app-specific content fields, which is very complex. 
In the meantime, you can check out our [Horizon repo](https://github.com/swellstores/horizon). This is currently in beta, and we're still fixing some last-minute bugs, but it should be ready to be explored. Everything is built in React and Next.js. 

Let us know if you have any other questions 😄
